modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -80,15 +80,21 @@ nodePlacement:
 ----
 ====
 
-//|`tlsSecurityProfile`
-//|`tlsSecurityProfile` specifies settings for TLS connections for Ingress controllers.
-//
-//If not set, the default is based on the `apiservers.config.openshift.io/cluster` resource.
-//
-//When using the `Old`, `Intermediate`, and `Modern` profile types, the effective profile configuration is subject to change between releases. For example, given a specification to use the `Intermediate` profile deployed on release `X.Y.Z`, an upgrade to release `X.Y.Z+1` may cause a new profile configuration to be applied to the Ingress controller, resulting in a rollout.
-//
-//The minimum TLS version for Ingress controllers is `1.1`, and the maximum TLS version is `1.2`.  An implication of this restriction is that the Modern TLS profile type cannot be used because it requires TLS `1.3`.
-//
+|`tlsSecurityProfile`
+|`tlsSecurityProfile` specifies settings for TLS connections for Ingress controllers.
+
+If not set, the default value is based on the `apiservers.config.openshift.io/cluster` resource.
+
+When using the `Old`, `Intermediate`, and `Modern` profile types, the effective profile configuration is subject to change between releases. For example, given a specification to use the `Intermediate` profile deployed on release `X.Y.Z`, an upgrade to release `X.Y.Z+1` may cause a new profile configuration to be applied to the Ingress controller, resulting in a rollout.
+
+The minimum TLS version for Ingress controllers is `1.1`, and the maximum TLS version is `1.2`.
+
+[IMPORTANT]
+====
+The `Modern` profile is not supported because it requires TLS `1.3`. The Ingress Operator converts the `Modern` profile to `Intermediate`.
```

Comment:
> The `Modern` profile is not supported because it requires TLS `1.3`

We may want to provide additional information here. The current HAProxy ingress controller (i.e. router) image does not support TLS 1.3.

Reply:
Thanks! Added.